mm2src/mm2_main/src/lp_ordermatch.rs

```
@@ -5221,6 +5221,30 @@ pub enum CancelBy {
     Coin { ticker: String },
 }
 
+pub async fn get_matching_orders(ctx: &MmArc, ticker: &str) -> Vec<Uuid> {
+    let mut matching_orders = vec![];
+    if let Ok(ordermatch_ctx) = OrdermatchContext::from_ctx(ctx) {
```

Why not return an error instead of using `if let`